src/CoreWCF.WebHttp/src/CoreWCF/UriTemplateMatchException.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.Serialization;
+
+namespace CoreWCF
+{
+    [Serializable]
+    public class UriTemplateMatchException : SystemException
+    {
+        public UriTemplateMatchException()
+        {
+        }
+        public UriTemplateMatchException(string message)
+            : base(message)
```

Comment:
Missing blank lines

Reply:
Addressed in 14ac88e67a220d805d2fac57fa31026cd397ba1e.